lib/rihanna.ex

```diff
@@ -49,20 +51,19 @@ defmodule Rihanna do
   > Rihanna.enqueue(MyJob, arg)
   """
 
+  def enqueue(term, opts \\ %{})
+
   @doc """
   Enqueues a job specified as a simple mod-fun-args tuple.
 
   ## Example
 
       > Rihanna.enqueue({IO, :puts, ["Umbrella-ella-ella"]})
   """
-  @spec enqueue({module, atom, list()}) :: {:ok, Rihanna.Job.t()}
-  def enqueue(term = {mod, fun, args}) when is_atom(mod) and is_atom(fun) and is_list(args) do
-    Rihanna.Job.enqueue(term)
-  end
-
-  def enqueue(_) do
-    raise ArgumentError, @enqueue_help_message
+  @spec enqueue({module(), atom(), list()}, map()) :: {:ok, Rihanna.Job.t()}
+  def enqueue(term = {mod, fun, args}, opts)
```

Comment:
Why not make opts an optional argument?

Reply:
It is optional.
The header: `def enqueue(term, opts \\ %{})`